modules/internal/ChapelTaskData.chpl

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2004-2017 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// ChapelTaskData.chpl
+//
+module ChapelTaskData {
+
+  use ChapelStandard;
+
+  // use ChapelError
+  // use CPtr
+
+  private extern proc sizeof(type t):size_t;
+  private extern proc memcpy(dst:c_ptr, src:c_ptr, n:size_t);
```

Comment:
I originally did but I'd rather not tie these two together in that way if we don't have to.